README.md

```diff
@@ -93,6 +93,34 @@ Building clang from sources takes time and resources and it can be avoided:
   moment, see [KhronosGroup/SPIRV-LLVM-Translator#477](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/477)
   to track progress, discuss and contribute.
 
+### Build with SPIRV-Tools
+
+The translator can use SPIRV-Tools to generate assembly with conformant syntax.
+If SPIRV-Tools have been installed prior to the build it will be detected and
+used automatically. However it is also possible to enable use of SPIRV-Tools
+from a custom location using the following instructions:
+
+1. Checkout, build and install SPIRV-Tools using
+   [the following instructions](https://github.com/KhronosGroup/SPIRV-Tools#build).
+   Example using cmake:
+```
+cmake -G Ninja .. -DCMAKE_INSTALL_PREFIX=<location to installation>
+ninja install
+```
+2. Using pkg-config when building the translator by setting
```

Comment:
```suggestion
2. Use pkg-config when building the translator by setting
```

Reply:
I am not sure what part is not correct - it is saying that by means of pkg-config we can specify the SPIRV-Tools installation...

Reply:
Maybe rephrase it as:

```suggestion
2. Point pkg-config to the SPIR-V tools installation when configuring the translator by setting
```